ci/jobs/projects.yaml

```diff
@@ -0,0 +1,110 @@
+- project:
+    name: build-automation
+    jobs:
+     - 'build-automation-repo-{release_config}'
+    release_config:
+     - 2.6-dev
+     - 2.6-testing
+     - 2.6-release
+     - 2.7-dev
+     - 2.8-dev
+     - master
+
+- project:
```

Comment:
Not sure if I like this way of defining jobs, since we are "duplicating" code. Creating a project for an actual job definition seems reinventing the wheel because there is no practical difference of having a job definition. On the other hand projects are required for job-template definitions and in this case required to exist.